src/video_emotion/api_helper.py

```diff
@@ -14,6 +33,28 @@ def process_data_and_extract_emotions(video_id, video_path, time_range):
 
 def process_data_and_extract_emotions_with_song(video_id, video_path, 
                                                 time_range, song_id):
+    """Clasifies the emotions in a video and places it in the database with a songid
```

Comment:
Typo here. Should probably also be "Classifies the emotions in the given video and places it in the database with a songid"